src/Services/FormSercive.js

```diff
@@ -1,50 +1,50 @@
-// export const FormService = (formParentElement, onSubmit) => {
-//     const form = formParentElement.querySelector('form');
+export const FormService = (formParentElement, onSubmit) => {
+    const form = formParentElement.querySelector('form');
 
-//     function validateForm() {
-//         let isFormValid = true;
+    function validateForm() {
+        let isFormValid = true;
 
-//         // Validate each form field
-//         for (const input of form.elements) {
-//             if (input.type === 'text' && input.value.trim().length < 2) {
-//                 isFormValid = false;
-//                 input.nextElementSibling.textContent = 'This field is required and must be at least 2 characters';
-//                 input.nextElementSibling.style.color = 'red';
-//             } else if (input.type === 'number' && isNaN(input.value)) {
-//                 isFormValid = false;
-//                 input.nextElementSibling.textContent = 'This field must be a number';
-//                 input.nextElementSibling.style.color = 'red';
-//             } else if (input.type === 'email' && !/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(input.value)) {
-//                 isFormValid = false;
-//                 input.nextElementSibling.textContent = 'This field must be a valid email';
-//                 input.nextElementSibling.style.color = 'red';
-//             }
-//         }
-//         return isFormValid;
-//     }
+        // Validate each form field
+        for (const input of form.elements) {
+            if (input.type === 'text' && input.value.trim().length < 2) {
+                isFormValid = false;
+                input.nextElementSibling.textContent = 'This field is required and must be at least 2 characters';
+                input.nextElementSibling.style.color = 'red';
+            } else if (input.type === 'number' && isNaN(input.value)) {
+                isFormValid = false;
+                input.nextElementSibling.textContent = 'This field must be a number';
+                input.nextElementSibling.style.color = 'red';
+            } else if (input.type === 'email' && !/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(input.value)) {
+                isFormValid = false;
+                input.nextElementSibling.textContent = 'This field must be a valid email';
+                input.nextElementSibling.style.color = 'red';
+            }
+        }
```

Comment:
looks like you returning same convention, 
what happened if you will have more inputs and more validations